rust/src/storage/s3/dynamodb_lock.rs

```diff
@@ -23,6 +23,8 @@ mod options {
     pub const REFRESH_PERIOD_MILLIS: &str = "DYNAMO_LOCK_REFRESH_PERIOD_MILLIS";
     /// Environment variable for `additional_time_to_wait_for_lock` option.
     pub const ADDITIONAL_TIME_TO_WAIT_MILLIS: &str = "DYNAMO_LOCK_ADDITIONAL_TIME_TO_WAIT_MILLIS";
+    /// Environment variable for `delete_data_on_release` option.
+    pub const DELETE_DATA_ON_RELEASE: &str = "DYNAMO_DELETE_DATA_ON_RELEASE";
```

Comment:
Do we need this env flag? Would it be possible to remove the added complexity and assume we will always delete data on release?

Reply:
Yes, moreover, when thinking more about that since we don't need a data fields after the lock is released, we might just delete lock on release, as in one of the option in reference impl